packages/lib/CalendarService.ts

```diff
@@ -259,14 +275,14 @@ export default abstract class BaseCalendarService implements Calendar {
   }
 
   isValidFormat = (url: string): boolean => {
-    const acceptedFormats = ["eml", "ics"];
-    const urlFormat = url.split(".").pop();
```

Comment:
Not a fan of this rename, formats is a better descriptor than extension - NIT - in what scenario did this not work (what is the fail case?)